pkg/plugin/plugin.go

```diff
@@ -264,3 +264,12 @@ func (p *plugin) ForceUpdate(ctx context.Context, cs *api.OpenShiftManagedCluste
 	p.log.Info("force updates successful")
 	return nil
 }
+
+func (p *plugin) GetPluginVersion(ctx context.Context, pluginTemplate *pluginapi.Config) ([]byte, error) {
```

Comment:
The version is in the config struct.  Where does that get stored?  How does this get the pluginapi.confg?  Are we storing that somewhere other than the config blob?